src/app/shared/state/entry-wizard.service.ts

```diff
@@ -117,7 +117,7 @@ export class EntryWizardService {
     this.entryWizardStore.setLoading(true);
     const registryEnum = this.convertSourceControlStringToEnum(repository.gitRegistry);
     this.workflowsService
-      .deleteWorkflow(registryEnum, repository.organization, repository.repositoryName)
```

Comment:
Not sure I understand this fix. Do you have a link to the broken build?

Reply:
https://app.circleci.com/pipelines/github/dockstore/dockstore-ui2/10979/workflows/60ec99da-8d68-427a-a15c-b15fd7164f27/jobs/49176